test/e2e/scenarios/ACL/acl_create_update.yaml

```diff
@@ -80,6 +80,10 @@ steps:
           ACK.ResourceSynced:
             status: "True"
             timeout: 180
+    expect:
```

Comment:
We will want assertions for AWS after every step. After the create, check the resource was actually created in memorydb. Check the important parts of the spec match the created resource. Especially check that any part of the spec that uses custom code was created/updated properly.

Reply:
I think if a resource is synced which means the resource exists and is created successfully. Correct me if I'm wrong. I can add spec check for resource creations if you think that's necessary~

Reply:
Added.